examples/210-embedding_models.ipynb

```diff
@@ -0,0 +1,1242 @@
+{
```

Comment:
<p>We have <code>is_freezed</code> property. Why are you suggesting to use private attribute? </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/etna-team/etna/pull/304/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
fixed